anesthetic/samples.py

```diff
@@ -132,7 +140,8 @@ def plot(self, ax, paramname_x, paramname_y=None, *args, **kwargs):
             nsamples = 500
             plot = scatter_plot_2d
         else:
-            raise ValueError("plot_type must be in {'kde', 'scatter'}")
+            raise ValueError("plot_type is '%s', but must be in "
+                             "{'kde', 'scatter'}." % plot_type)
```

Comment:
It's probably more consistent to call this a NotImplementedError

Reply:
Agreed, but was hesitant to overwrite your code...

Reply:
done

Reply:
Don't be -- PRs are a good opportunity to clean up inconsistencies